src/app/pages/all-tests/tests-list/test-item/test-item.component.scss

```diff
@@ -1,4 +1,4 @@
-@import "../../../../shared/colors";
+@import "../../../../../assets/styles/colors";
```

Comment:
so ugly relative path :(  Can we do absolute path, like this `/assets/styles/colors` ?

Reply:
fixed with 'src/assets/styles/colors'